src/sections/Company/Brand/Brand-components/servicemeshpatterns.js

```diff
@@ -67,7 +67,7 @@ const ServiceMeshPatterns = () => {
         </Col>
         <Col>
           <p>
-                            The Service Mesh Pattern color palette consists of gray, green and colors and
+                            The Service Mesh Pattern color palette consists of gray, green and blue colors along with
```

Comment:
we don't use blue in service-mesh-patterns, just green and gray.

Reply:
Thanks for reviewing @warunicorn19 . So shall I frame it this way : `The Service Mesh Pattern color palette consists of gray and green colors along with .. ` ?

Reply:
Yup, sounds good👍